### ISSUE TITLE
Potentially false information in: Where does Azure DevOps Services get your name and email?

### ISSUE THREAD

**Issue by Merlin2001** [MicrosoftDocs/azure-devops-docs#11654]
### Issue
The section _Where does Azure DevOps Services get your name and email?_ doesn't seem to match the currently implemented behavior. According to the section, "Azure Repos supplies your profile details as the author of the commit", but from my testing, there is no way to change the details that will be used for the commit.

### Setup
The original account info of the AZD account is `User Name <uname@companysolutions.com>`, my local commits are done using `User Name <uname@company.com>`.  As I'm in a corporate environment I assume my account is an AD account (same account to login to Windows, Teams etc.)

### Steps to reproduce
Here is what I tried and found out:  
I changed the author name and email address in the local git config as well as on the [my information](https://aex.dev.azure.com/me) page online

* All commits made via the AZD web interface showed `User Name <uname@companysolutions.com>` **regardless** of what I set in my information
* For all local git commits, the author name as given by the commit was shown also in AZD
* All commits made by `<uname@companysolutions.com>` showed my profile picture regardless of the author name (i.e. it didn't have to match the "Full Name" set in [my information](https://aex.dev.azure.com/me) 

In summary there seems to be no way to alter commit info in the web interface.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 63d13b7b-c365-5157-8b7d-f2d01b4c8650
* Version Independent ID: 780701ea-4a09-9c25-98cd-4c41bcfcf33d
* Content: [Author names in Git - Azure Repos](https://docs.microsoft.com/en-us/azure/devops/repos/git/git-names?view=azure-devops)
* Content Source: [docs/repos/git/git-names.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/repos/git/git-names.md)
* Product: **devops**
* Technology: **devops-code-git**
* GitHub Login: @vijayma
* Microsoft Alias: **vijayma**
